pkgs/development/python-modules/ffmpy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, ffmpeg
+}:
+
+buildPythonPackage rec {
+  pname = "ffmpy";
+  version = "0.3.0";
+
+  disabled = pythonOlder "3.6";
+
+  # The github repo has no release tags, the pypi distribution has no tests.
+  # This package is quite trivial anyway, and the tests mainly play around with the ffmpeg cli interface.
+  # https://github.com/Ch00k/ffmpy/issues/60
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "dXWRWB7uJbSlCsn/ubWANaJ5RTPbR+BRL1P7LXtvmtw=";
+  };
+
+  propagatedBuildInputs = [
+    ffmpeg
+  ];
```

Comment:
https://github.com/Ch00k/ffmpy/blob/master/ffmpy.py#L14

Instead of propagating ffmpeg, replace the executable path here with the absolute path to the ffmpeg executable